repos.d/gentoo/liguros.yaml

```diff
@@ -1,39 +1,37 @@
 ###########################################################################
 # Liguros
 ###########################################################################
-- name: liguros_20_7
+- name: liguros_21_7
   type: repository
-  desc: LiGurOS 20.7
+  desc: LiGurOS 21.7
   statsgroup: Gentoo
   family: gentoo
   ruleset: [gentoo, liguros]
   color: '1de9b6'
   minpackages: 28000
+  valid_till: 2022-02-01
   default_maintainer: fallback-mnt-liguros@repology
   sources:
     - name:
```

Comment:
Since there's now only a single source, please turn this into a single value instead of a list, e.g. `- name: liguros-repo`